requirements.txt

```diff
@@ -7,3 +7,4 @@ dataclasses-json==0.6.4
 torch==2.3.1
 torchvision
 tabulate==0.9.0
+torchmetrics
```

Comment:
@colon3ltocard do we need to add versions to each package ?

Reply:
Yes